helm/cloud-provider-azure/Chart.yaml

```diff
@@ -13,4 +13,4 @@ maintainers:
 name: cloud-provider-azure
 sources:
 - https://github.com/kubernetes-sigs/cloud-provider-azure
-version: 1.28.1
+version: 1.28.2
```

Comment:
@jackfrancis Should this not bump the chart version if it doesn't change the kubernetes version?

Reply:
https://github.com/kubernetes-sigs/cloud-provider-azure/tree/master/helm/cloud-provider-azure#how-to-pick-an-appropriate-new-version-when-updating-the-helm-chart